doc/prop.xml

```diff
@@ -788,3 +788,38 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="IsHamiltonianDigraph">
+<ManSection>
+  <Prop Name="IsHamiltonianDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is Hamiltonian, this property returns
+    <K>true</K>.<P/>
+
+    A Hamiltonian digraph is a digraph which has a circuit visiting 
```

Comment:
"<E>Hamiltonian</E>" and wouldn't it be better to write, something like "A digraph is <E>Hamiltonian</E>..."?